hack/e2e/run.sh

```diff
@@ -116,6 +118,15 @@ if [[ -r "${EBS_SNAPSHOT_CRD}" ]]; then
   kubectl apply -f "$EBS_SNAPSHOT_CRD"
   # TODO deploy snapshot controller too instead of including in helm chart
 fi
+endSec=$(date +'%s')
```

Comment:
have to wait for the pods to become ready. not so easy in bash to be honest.

Reply:
Yeah agree, I did some research but no luck to find any tool to track container start up time.
But one thing is if we use helm --wait flag, helm will wait for containersReady condition before exit. IMO It would be good enough as for now.

Reply:
And open for any suggestion to track this info :-)

Reply:
ooh yeah let's use helm built-in functionality --wait flag, that is cool if they have it. 

Otherwise I was thinking of how to call a python/go script from here and use the kube python/go  client, which would be really ugly...

Reply:
Maybe we can use cloudwatch? The tests run in an AWS-internal account. And if running locally, i don't mind pushing metrics to my own account. We can fail-open in case for whatever reason the metric push fails (transient cloudwatch issue or whatever)

Reply:
That's a good find. Does it wait for everything deployed by the chart to be ready?

Reply:
Ideally there is a way to expose the metric publicly though, cloudwatch wont be visible like https://testgrid.k8s.io/provider-aws-efs-csi-driver#e2e-test&width=20 is.

Reply:
Yes, it will wait for all the containers to be ready